src/libs/actions/App.js

```diff
@@ -211,23 +211,17 @@ function setUpPoliciesAndNavigate(session) {
         return;
     }
     if (!isLoggingInAsNewUser && exitTo) {
-        if (Navigation.isDrawerRoute(exitTo)) {
-            // The drawer navigation is only created after we have fetched reports from the server.
-            // Thus, if we use the standard navigation and try to navigate to a drawer route before
-            // the reports have been fetched, we will fail to navigate.
-            Navigation.isDrawerReady()
-                .then(() => {
-                    // We must call dismissModal() to remove the /transition route from history
-                    Navigation.dismissModal();
-                    Navigation.navigate(exitTo);
-                });
-            return;
-        }
+        // The drawer navigation is only created after we have fetched reports from the server.
```

Comment:
NAB: This comment feels a bit out of place here, since the next line is checking if the Navigation as a whole is ready

Reply:
I can move it tomorrow! Just waiting to see if there is something else I should change and I can change all together.